library/std/src/sys/args/mod.rs

```diff
@@ -0,0 +1,43 @@
+//! Platform-dependent command line arguments abstraction.
+
+#![forbid(unsafe_op_in_unsafe_fn)]
+
+#[allow(dead_code)]
+mod common;
```

Comment:
I'm not a big fan of an unconditional `#[allow(dead_code)]`. Could you perhaps move the module definition to the individual platforms, or alternatively use `cfg_attr` to enable the lint on at least one platform (preferably Linux)?

Reply:
`#[path]`-based imports have a predisposition to break, but I think it's fine in this case.